### ISSUE TITLE
Add additional information to revision history

### ISSUE THREAD

**Issue by rjp2525** [VentureCraft/revisionable#185]
This package is fantastic, but there's one thing that it doesn't have that I needed.

For those of us using this package for security precautions like me, we'd like to store more information about the request at the time of the revision. (IP address, user agent, etc.)

I ended up implementing it somewhat messy within my application, so it'd be great if it was part of the package by default. If I find some time I'll send in a PR.

**Comment by Feld0** [comment]
+1. IP address logging will make this an even more useful package for auditing purposes.

**Comment by martinmarianetti** [comment]
+1

**Comment by ns3777k** [comment]
+1

**Comment by TheRinseM** [comment]
+1, @rjp2525  Can you explain how you ended up implementing this yourself?

**Comment by Perturbatio** [comment]
@astierler in https://github.com/VentureCraft/revisionable/pull/258#issuecomment-271775300 you mentioned implementing meta already in another way, is there any chance you'll be adding that option to the package?  Or make revisionable modifiable in such a way that it can be added as a separate package?

**Comment by astierler** [comment]
@Perturbatio yes, I'll resume putting together a new major version this month/in January at latest (in middle of move now) - which will allow pre and post hooks to modify data/records going into DB per model, if that seems like the best way (allows the most customization) - just need to think of how to address the schema of this going forward if any changes are done there!
